packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -414,32 +424,39 @@ pub async fn register_indexer(
     namespace: &str,
     identifier: &str,
     pubkey: Option<&str>,
+    created_at: DateTime<Utc>,
 ) -> sqlx::Result<RegisteredIndex> {
     if let Some(index) = indexer_is_registered(conn, namespace, identifier).await? {
         return Ok(index);
     }
 
     let row = sqlx::query(
-        "INSERT INTO index_registry (namespace, identifier, pubkey)
-         VALUES ($1, $2, $3)
+        "INSERT INTO index_registry (namespace, identifier, pubkey, created_at)
+         VALUES ($1, $2, $3, $4)
          RETURNING *",
     )
     .bind(namespace)
     .bind(identifier)
     .bind(pubkey)
+    .bind(created_at)
     .fetch_one(conn)
     .await?;
 
     let id: i64 = row.get(0);
     let namespace: String = row.get(1);
     let identifier: String = row.get(2);
     let pubkey = row.get(3);
+    let created_at: DateTime<Utc> = {
+        let created_at: NaiveDateTime = row.try_get("created_at")?;
+        DateTime::<Utc>::from_utc(created_at, Utc)
+    };
```

Comment:
Are we able to use `row.get(4)` here to remain consistent with your implementation on lines 476-479?